src/libhirte/common/cfg.c

```diff
@@ -8,10 +8,11 @@
 #include <string.h>
 #include <unistd.h>
 
+#include <hashmap.h>
```

Comment:
Don't you need the `NOLINTNEXTLINE` comment here? And if not, do you need it in `agent.h`?